### PR TITLE
Configure wheels Fixes #43

### LINKED ISSUE #43: release wheels.

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    url='https://github.com/techtonik/python-patch/',
+    description='Patch utility to apply unified diffs',
+    license='MIT',
+    py_modules=['patch'],
+    long_description=readme,
+    classifiers=[
+        "Environment :: Console",
+        "License :: OSI Approved :: MIT License",
+        "Operating System :: OS Independent",
+        "Programming Language :: Python",
```
Comment (graingert) [inline setup.py]: This also adds missing trove classifiers

**Comment by techtonik** [comment]
Well, thanks for taking action to add wheels support, but don't you think that this adds too much meaningless garbage and duplicate info to the project?

I created `pypack` in the first place to keep the code clean and maintenance minimal. So if you want to improve the state of art, I suggest to add wheel support to `pypack`. The process is described here - https://leanpub.com/reversingpythonpackaging along with ready to use code snippets.

But note the this will require creating tests on wheel packing, unpacking, to make the tool future proof against `pip` purists, who want to complicate the wheel installation process with (again) poorly argumented per file signature checks (which are easy to forge) etc.

**Comment by graingert** [comment]
> don't you think that this adds too much meaningless garbage and duplicate info to the project?

I don't think that at all. None of this is duplicated

> I created pypack

This PR obviates the need for this extra level of indirection ;)

**Comment by graingert** [comment]
>  Reasons why Python packaging suxx

Hmm ok...

**Comment by graingert** [comment]
> And also you need to know, that I was expelled from Python community, because I didn't want to follow the lemmings way as I put it. 

I consider myself a member of this community. Thanks for pointing this out. Maybe you should add this to your README.md?

**Pull request closed without merging**